app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,18 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+            self,
+            nickname: str,
+            musical_instrument: str,
+            bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self.bow_level = bow_level
```

Comment:
The `bow_level` attribute should be stored as a protected attribute, i.e., `_bow_level`, to align with the task requirement of encapsulating additional parameters as protected attributes.